dendrite/pipeline.yml

```diff
@@ -37,6 +37,7 @@ steps:
           mount-buildkite-agent: false
 
   - command:
+      - docker build -t complement-dendrite -f build/scripts/Complement.Dockerfile .
       - "wget -N https://github.com/matrix-org/complement/archive/master.tar.gz && tar -xzf master.tar.gz"
       - "cd complement-master && COMPLEMENT_BASE_IMAGE=complement-dendrite:latest go test -v ./tests"
```

Comment:
I would say that at the point you have five commands to run, you should consider putting a script in the `dendrite` repo and using that instead.

but this is fine for now.

Reply:
It was originally a script in the Dendrite repo but we decided against it because it would then be under the control of anyone in a PR.